nconf/metrics.go

```diff
@@ -13,23 +13,25 @@ import (
 
 type MetricsConfig struct {
 	DataDog *struct {
-		APIKey string `mapstructure:"api_key"`
-		AppKey string `mapstructure:"app_key"`
+		APIKey string `mapstructure:"api_key" split_words:"true"`
+		AppKey string `mapstructure:"app_key" split_words:"true"`
 	} `mapstructure:"datadog"`
 
-	SFXToken string `mapstructure:"sfx_token"`
+	SFXToken string `mapstructure:"sfx_token" split_words:"true"`
```

Comment:
I'm not 100% sure what will happen with `split_words` here. It might be `S_F_X_TOKEN` 😅 .